tools/docker/ubuntu18.04-ci-cpu.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gluonai/gluon-nlp:cpu-base-latest
```

Comment:
There may be a problem with relying on `gluonai/gluon-nlp:cpu-base-latest` here and in the other images below:

1) the image is pushed manually and only time-to-time by developers. It won't get any security updates and changes to the upstream (`ubuntu` / `nvidia`) containers
2) the system state is not controlled via source file, as it relies on the extrenal dockerhub state managed by developers separately from the source code

Instead, you can consider using a single [Dockerfile above with multiple stages](https://docs.docker.com/develop/develop-images/multistage-build/#use-a-previous-stage-as-a-new-stage)

Reply:
Looks good!